crates/plugins/mediator-coordination/src/plugin.rs

```diff
@@ -1,17 +1,20 @@
 use axum::Router;
 use keystore::filesystem::StdFileSystem;
 use plugin_api::{Plugin, PluginError};
-use mongodb::{error::Error as MongoError, options::ClientOptions, Client, Database};
+use mongodb::{ options::ClientOptions, Client, Database};
 use std::sync::Arc;
 
 use crate::{
     repository::stateful::coord::{MongoConnectionRepository, MongoSecretsRepository},
     util,
     web::{self, AppState, AppStateRepository},
 };
-
 #[derive(Default)]
-pub struct MediatorCoordinationPlugin;
+pub struct MediatorCoordinationPlugin {
+    env: Option<MediatorCoordinationPluginEnv>,
+    db: Option<Database>,
+}
+
 
 struct MediatorCoordinationPluginEnv {
     _public_domain: String,
```

Comment:
It is now okay to remove the leading underscore.
```suggestion
    public_domain: String,
```